content-scripts/inject/module.js

```diff
@@ -12,7 +12,7 @@ scratchAddons.eventTargets = {
 scratchAddons.session = {};
 const consoleOutput = (logAuthor = "[page]") => {
   const style = {
-    // Remember to change these as well on cs.js
+    // Remember to change these as well on cs-console.js
```

Comment:
cant we make this an external lib already

Reply:
Good point, we now could